ansible/playbooks/AdoptOpenJDK_Unix_Playbook/trss.yml

```diff
@@ -0,0 +1,157 @@
+          tags: hostname
+
+        - name: OS update -- apt-get upgrade
+          apt: upgrade=safe update_cache=yes
+          tags: patch_update
+
+        - name: Add Node.js v12.x
+          raw: "curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -"
+          tags: dependencies
+
+        - name: Import MongoDB 4.2 key
+          raw: "wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | sudo apt-key add -"
+          tags: mongodb
+
+        - name: Create list file for MongoDB
+          raw: "echo \"deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.2 multiverse\" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.2.list"
+          tags: mongodb
+
+        - name: Install API prerequisistes
+          apt:
+            name: "{{ item }}"
+            update_cache: yes
+          with_items:
+            - git
+            - nodejs
+            - mongodb-org
+            - make
+            - build-essential
+            - g++
+          tags:
+            - dependencies
+            - skip_ansible_lint
+
+        - name: Start mongoDB
+          systemd:
+            name: mongod
+            state: started
+          tags: mongodb
+
+        - name: Create Jenkins user
+          action: user name={{ Jenkins_Username }} state=present shell=/bin/bash
+          ignore_errors: yes
+          tags: jenkins_user
+
+        - name: Install NPM packages globally
+          npm:
+            name: "{{ item }}"
+            global: yes
+            state: present
+          with_items:
+            - forever
+            - forever-service
+            - yarn
+          tags: forever
+
+        - name: Git Clone openjdk-test-tools.git
+          git:
+            repo: 'https://github.com/AdoptOpenJDK/openjdk-test-tools.git'
+            dest: "/home/{{ Jenkins_Username }}/openjdk-test-tools"
+            update: yes
+            force: yes
+
+        - name: Change owner permissions
+          file:
+            path: /home/{{ Jenkins_Username }}/openjdk-test-tools
+            owner: "{{ Jenkins_Username }}"
+            group: "{{ Jenkins_Username }}"
+
+        - name: npm ci && npm run build of test-result-summary-client
+          shell: cd /home/{{ Jenkins_Username }}/openjdk-test-tools/test-result-summary-client/ && npm ci && npm run build
+
+        - name: npm ci of TestResultSummaryService
+          shell: cd /home/{{ Jenkins_Username }}/openjdk-test-tools/TestResultSummaryService/ && npm ci
+
+        - name: Install nginx seperately
+          apt:
+            name: nginx
+            state: present
+          tags: nginx_conf
+          register: nginx_installed
+
+        - name: Add app root directorty to /etc/nginx/sites-enabled/default
+          lineinfile:
+            path: /etc/nginx/sites-enabled/default
+            regex: 'root /var/www/html;'
+            line: 'root /home/{{ Jenkins_Username }}/openjdk-test-tools/test-result-summary-client/build/;'
```

Comment:
I presume the username gets expanded ok in there with the single quotes around it?

Reply:
Yes. The saved line becomes `root /home/jenkins/openjdk-test-tools/test-result-summary-client/build/;`